python/ray/air/_internal/session.py

```diff
@@ -4,7 +4,7 @@
 logger = logging.getLogger(__name__)
 
 
-def _get_session(warn: bool = True):
+def _get_session(warn: bool = False):
```

Comment:
We already log a separate warning elsewhere -- no need for this message that talks about a "tune vs train session"